nemo/core/optim/lr_scheduler.py

```diff
@@ -493,6 +502,52 @@ def _noam_annealing(self, initial_lr, step):
         return out_lr
 
 
+class NoamHoldAnnealing(WarmupHoldPolicy):
+    def __init__(self, optimizer, *, max_steps, decay_rate=0.5, min_lr=0.0, last_epoch=-1, **kwargs):
+        """
+        Implementation of the Noam Hold Annealing policy from the SqueezeFormer paper.
```

Comment:
Should we have more detail on how it works?

Reply:
Yes good point.